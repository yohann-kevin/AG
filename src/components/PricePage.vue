<template>
  <div class="page-prices">
    <h2>Nos tarifs</h2>
    <v-card
      elevation="8"
      shaped
      class="pack-sample"
    >
      <v-card-title>
        <v-icon
          large
          left
        >
          mdi-star-check
        </v-icon>
        <span class="text-h6 font-weight-light">Pack simple</span>
      </v-card-title>

      <v-card-text class="font-weight-bold">
        <div class="my-4 text-subtitle-1">
          100 €
        </div>
        Vous pourrez prendre contact avec 5 modèles d'AG Scouting.
      </v-card-text>

      <v-btn color="black lighten-2" text class="pack-btn">
        Bientôt disponible
      </v-btn>
    </v-card>


    <v-card
      elevation="8"
      shaped
      class="pack-gold"
    >
      <v-card-title>
        <v-icon
          large
          left
          class="gold-star"
        >
          mdi-star-check
        </v-icon>
        <span class="text-h6 font-weight-light">Pack gold</span>
      </v-card-title>

      <v-card-text class="font-weight-bold">
        <div class="my-4 text-subtitle-1">
          300 €
        </div>
        Vous pourrez prendre contact avec 25 modèles d'AG Scouting.
      </v-card-text>

      <v-btn color="black lighten-2" text class="pack-btn">
        Bientôt disponible
      </v-btn>
    </v-card>
  </div>
</template>

<script>
  export default {
    name: 'PricePage',

    data: () => ({
    }),
  }
</script>

<style scoped>
.page-prices {
  width: 100%;
  display: flex;
  justify-content: space-around;
  flex-wrap: wrap;
}

.page-prices h2 {
  width: 100%;
  text-align: center;
  margin: 15px;
}

.pack-sample,
.pack-gold {
  width: 35%;
}

.gold-star {
  color: #FFD700;
}

.pack-btn {
  margin: 10px;
}

@media only screen and (max-width: 768px) {
  .pack-sample,
  .pack-gold {
    width: 80%;
    margin-bottom: 35px;
  }
}
</style>
