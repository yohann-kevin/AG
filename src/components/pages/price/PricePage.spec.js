import { shallowMount, createLocalVue } from '@vue/test-utils';
import Vuetify from 'vuetify';
import PricePage from './PricePage.vue';

describe('ResetPassword', () => {
  const initWrapper = () => {
    const localVue = createLocalVue();
    const vuetify = new Vuetify();
    return shallowMount(PricePage, {
      localVue,
      vuetify
    });
  };

  it('should test component render correctly', () => {
    const wrapper = initWrapper();

    const title = wrapper.find('h2');
    expect(title.text()).toContain('Nos tarifs');
    expect(wrapper).toMatchSnapshot();
  });
});
