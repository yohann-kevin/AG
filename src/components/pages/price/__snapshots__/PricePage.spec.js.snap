// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResetPassword should test component render correctly 1`] = `
<div class="page-prices">
  <h2>Nos tarifs</h2>
  <v-card elevation="8" shaped="" class="pack-bronze">
    <v-card-title>
      <v-icon large="" left="" class="bronze-star">
        mdi-star-check
      </v-icon> <span class="text-h6 font-weight-light">Modèle Bronze</span>
    </v-card-title>
    <v-card-text class="font-weight-bold">
      <div class="my-4 text-subtitle-1">
        T7 : 98,55 €
      </div>
      Les modèles T7 sont tous les modèles qui n'ont encore jamais travaillé sur AG Scouting.
    </v-card-text>
    <v-btn color="black lighten-2" text="" class="pack-btn">
      Bientôt disponible
    </v-btn>
  </v-card>
  <v-card elevation="8" shaped="" class="pack-silver">
    <v-card-title>
      <v-icon large="" left="" class="silver-star">
        mdi-star-check
      </v-icon> <span class="text-h6 font-weight-light">Modèle Silver</span>
    </v-card-title>
    <v-card-text class="font-weight-bold">
      <div class="my-4 text-subtitle-1">
        T8 : 114,80 €
      </div>
      Les modèles T8 ont déjà travaillé contre une rémunération avec AG Scouting pour un client.
    </v-card-text>
    <v-btn color="black lighten-2" text="" class="pack-btn">
      Bientôt disponible
    </v-btn>
  </v-card>
  <div class="page-prices-partie-2">
    <v-card elevation="8" shaped="" class="pack-gold">
      <v-card-title>
        <v-icon large="" left="" class="gold-star">
          mdi-star-check
        </v-icon> <span class="text-h6 font-weight-light">Modèle Gold</span>
      </v-card-title>
      <v-card-text class="font-weight-bold">
        <div class="my-4 text-subtitle-1">
          T9 : 151,15 €
        </div>
        Les T9 sont des modèles qui ont déjà travaillé à l'international pour des clients de AG Scouting.
      </v-card-text>
      <v-btn color="black lighten-2" text="" class="pack-btn">
        Bientôt disponible
      </v-btn>
    </v-card>
    <v-card elevation="8" shaped="" class="pack-diamond">
      <v-card-title>
        <v-icon large="" left="" class="diamond-star">
          mdi-star-check
        </v-icon> <span class="text-h6 font-weight-light">Modèle Diamond</span>
      </v-card-title>
      <v-card-text class="font-weight-bold">
        <div class="my-4 text-subtitle-1">
          T10 : 188,65 €
        </div>
        Les T10 sont des modèles reconnus et ayant déjà une forte expérience en France ou à l'international.
      </v-card-text>
      <v-btn color="black lighten-2" text="" class="pack-btn">
        Bientôt disponible
      </v-btn>
    </v-card>
  </div>
  <v-alert dense="" text="" elevation="15" type="warning" class="message-warning">
    Les frais de déplacement et d'hébergements seront à votre charge si nécessaire.
  </v-alert>
  <v-alert dense="" text="" elevation="15" type="warning" class="message-warning">
    Afin de maintenir la qualité de notre offre des frais de service sont à prévoir.
  </v-alert>
</div>
`;
